project-api/ballerina-projects/src/main/java/io/ballerina/projects/writers/BaloWriter.java

```diff
@@ -18,9 +18,39 @@
 
 package io.ballerina.projects.writers;
 
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
 import io.ballerina.projects.Package;
+import io.ballerina.projects.model.BallerinaToml;
+import io.ballerina.projects.model.BaloJson;
+import io.ballerina.projects.model.Dependency;
+import io.ballerina.projects.model.PackageJson;
+import io.ballerina.projects.model.PlatformLibrary;
+import io.ballerina.projects.model.adaptors.JsonCollectionsAdaptor;
+import io.ballerina.projects.model.adaptors.JsonStringsAdaptor;
+import io.ballerina.projects.writers.exceptions.NoPermissionException;
+import org.ballerinalang.compiler.BLangCompilerException;
+import org.ballerinalang.toml.model.Library;
+import org.wso2.ballerinalang.util.RepoUtils;
```

Comment:
We should use `io.ballerina.projects.utils.ProjectUtils`. Shall we copy the required utilities to this class from RepoUtils?

Reply:
Added to this class.